packages/plugin-protocol-http/src/http.ts

```diff
@@ -209,7 +214,7 @@ export default ({
                     method,
                     headers: { ...formHeaders, ...headers },
                     agent: customAgent,
-                    credentials: withCredentials ? 'include' : 'same-origin',
+                    credentials: credentials ?? withCredentials ? 'include' : 'same-origin',
```

Comment:
The `??` operator has higher priority so it will be executed first and the ternar operator condition.

See 
![image](https://github.com/Tinkoff/tinkoff-request/assets/8974488/67fc4d4d-f500-4469-9cc7-d8c36b8d6c8e)

Please, add grouping for condition to handle it properly

Reply:
Goood catch!